bindings/nodejs/examples/client/00_get_info.ts

```diff
@@ -1,11 +1,11 @@
-// Copyright 2021-2022 IOTA Stiftung
+// Copyright 2021-2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { Client, initLogger } from '@iota/client';
-require('dotenv').config({ path: '../.env' });
+import { Client, initLogger } from '@iota/sdk';
+require('dotenv').config({ path: '.env' });
 
 // Run with command:
-// node ./dist/00_get_info.js
```

Comment:
I am assuming you already tested that those example commands work and I can skip those? :trollface: